webrtc.html

```diff
@@ -866,7 +866,7 @@
 
                   <li>
                     <p>If <var>connection</var>'s <a>signaling state</a>
-                    changed above, fire a simple event named <code><a href=
+                    changed above, queue a task to fire a simple event named <code><a href=
```

Comment:
Note that all operations like "update connection's ICE gathering/connection state" are queued in the steps that are liked to.

The intention here is to have the state updated and receivers dispatched for all new incoming media before the promise "then-handler" is run. So signalingstatechange needs to be fired synchronously as the state is updated. The question is we need to the synchronous work later in the steps.